bandit/formatters/text.py

```diff
@@ -166,6 +166,14 @@ def report(manager, fileobj, sev_level, conf_level, lines=-1):
             "\tTotal lines skipped (#nosec): %i"
             % (manager.metrics.data["_totals"]["nosec"])
         )
+        bits.append(
+            "\tTotal lines skipped (#nosec BXXX,BYYY,...): %i"
```

Comment:
So, we're not skipping the entire line here. I think this is where I'm struggling with these changes. Should this instead say:

```
Total potential issues skipped due to specifically being disabled (e.g., #nosec BXXX): %i
```

I think that's what we mean

Reply:
Sure, that's what this is for. I'll change it.